test/ng/supportSpec.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
```

Comment:
It contains helpers for tests and their tests (when they exist) :smiley:
For lack of a better directory, keeping the pec file next to the source is better than putting it into an arbitrary directory 😛